server/cmd/usermatches/main.go

```diff
@@ -0,0 +1,257 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package main
+
+import (
+	"context"
+	"encoding/csv"
+	"encoding/hex"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+
+	"decred.org/dcrdex/dex"
+	"decred.org/dcrdex/server/account"
+	"decred.org/dcrdex/server/asset"
+	"decred.org/dcrdex/server/asset/btc"
+	"decred.org/dcrdex/server/asset/dcr"
+	"decred.org/dcrdex/server/asset/ltc"
+	"decred.org/dcrdex/server/db"
+	"decred.org/dcrdex/server/db/driver/pg"
+)
+
+// We do not need a Backend Setup, just the Drivers to call DecodeCoinID. While
+// we can do this with asset.DecodeCoinID(assetID, coinID), doing the following
+// drastically reduces the locking/unlocking (asset.driversMtx) that would be
+// required to decode coin IDs, and we will likely be doing many.
+var assets = map[uint32]asset.Driver{
+	0:  &btc.Driver{},
+	2:  &ltc.Driver{},
+	42: &dcr.Driver{},
```

Comment:
I know this will work, but if you wanted to add an exported getter in **server/asset/driver.go** for the package-registered `asset.Driver`, that'd be fine too, even if we're not using it in production.

Reply:
I'm ok with this I think, but a `asset.CoinDecoder(assetID uint32) func([]byte) (string, error)` that just returns the named asset's `(*Driver).DecodeCoinID` function might be good too.